src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,69 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int salaryFirst = 0;
+        int salarySecond = 0;
+        int salaryThird = 0;
+        String firstName = "";
```

Comment:
Remember about informative names of the variables and methods. Instead of generic names like 'firstName', 'secondName', and 'thirdName', use names that describe their purpose or role better.